Frontend/ui-common/src/Engineering/Helpers/Commons.js

```diff
@@ -308,6 +308,21 @@ export const getPastWeeks = function (count) {
   }
 };
 
+export const getPastMonths = function (count) {
+  try { 
+    const result = []
+    const currentDate = new Date(), month = currentDate.getMonth();
+    for(let i = 0; i < count; i++) {
+      const newDate = new Date(); newDate.setMonth(month - i);
+      const obj = { utcDate : newDate, month: month - i + 1};
+      result.push(obj);
+    }
+    return result.reverse();
+  } catch (e) {
+    console.log("error in getPastMonths", e);
+  }
+}
+
```

Comment:
Can you return empty array if it throws error? also why it needs try catch block?